presto-main/src/main/java/io/prestosql/operator/scalar/VarbinaryFunctions.java

```diff
@@ -279,6 +280,14 @@ public static Slice sha512(@SqlType(StandardTypes.VARBINARY) Slice slice)
         return computeHash(Hashing.sha512(), slice);
     }
 
+    @Description("Compute murmur3 hash")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice murmur3(@SqlType(StandardTypes.VARBINARY) Slice slice)
+    {
+        return Murmur3Hash128.hash(1, slice, 0, slice.length());
```

Comment:
Why seed = 1? Let's use the version with default seed:

```suggestion
        return Murmur3Hash128.hash(slice, 0, slice.length());
```

Reply:
With default seed, this method returns all zero bytes for empty string, thats why i added a seed.

Reply:
That's the expected hash for an empty string. For example, 

In python:
```
>>> import mmh3
>>> mmh3.hash('')
0
```

In Java:
```
System.out.println(Hashing.murmur3_32().hashBytes(new byte[0]).asInt());
-> 0
```